README.md

```diff
@@ -20,6 +20,19 @@ Additionally, depending on newer versions of Jenkins allows our builds to pick u
 These deprecations can often be migrated with new OpenRewrite recipes.
 Proactively handling deprecations in the plugins we depend on allows us to avoid runtime issues if and when the deprecated code is removed.
 
+## Quick start
+
+[Running Rewrite on a Maven project without modifying the build][mvn-cli] is very helpful for getting started.
+To run the `org.openrewrite.jenkins.github.AddTeamToCodeowners` recipe:
```

Comment:
At first glance there's a mismatch between the recipe here and further down.

Reply:
Thank you for the review, nice catch! I see Mark already fixed it  🎉